basilisk/gui/ocr_handler.py

```
@@ -312,21 +312,17 @@ def on_ocr(self, event: wx.CommandEvent):
-			"result_queue": result_queue,
-			"cancel_flag": cancel_flag,
 			"api_key": current_account.api_key.get_secret_value(),
 			"base_url": current_account.custom_base_url
 			or current_account.provider.base_url,
 			"attachments": attachment_files,
 			"log_level": log_level,
 		}
 
-		self.process = Process(
-			target=run_task, args=(engine.handle_ocr,), kwargs=kwargs
+		self.process = run_task(
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)